### PR TITLE
PMM-6695 grafana v7.3.7

### PR DESCRIPTION
**Pull request by lunaticusgreen**
**What this PR does / why we need it**: Upgrades grafana to version 7.3.7

**Which issue(s) this PR fixes**: PMM-6695

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: packages/grafana-ui/src/components/Slider/Slider.test.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ const sliderProps: Props = {
   max: 20,
 };
 
-describe('Slider', () => {
+xdescribe('Slider', () => {
```
[the remark this answers is not in the document]
Comment (lunaticusgreen) [reply packages/grafana-ui/src/components/Slider/Slider.test.tsx:10]: Thanks for the comment, I'll add it, the problem is slider tests started to fail without any visible changes of the slider, commented on it just to check the build working and not to block QA

**Comment by nicolalamacchia** [comment]
Is `master` the right branch this PR should be merged to?

File: public/app/features/integrated-alerting/components/AlertRules/AlertRules.utils.ts [hunk 2]
```diff
@@ -36,15 +36,15 @@ export const formatThreshold = (
...
 
   if (!thresholdParam) {
-    value = templateThresholdParam[type].default;
+    value = templateThresholdParam[type]?.default;
   } else {
     value = thresholdParam[type];
   }
 
-  const unit = AlertRulesListPayloadTemplateParamUnits[paramUnit];
+  const unit = paramUnit && AlertRulesListPayloadTemplateParamUnits[paramUnit];
```
[the remark this answers is not in the document]
Comment (lunaticusgreen) [reply public/app/features/integrated-alerting/components/AlertRules/AlertRules.utils.ts]: Thanks, fixed!

File: public/app/features/integrated-alerting/components/AlertRuleTemplate/AddAlertRuleTemplateModal/AddAlertRuleTemplateModal.tsx [hunk 3]
```diff
@@ -69,6 +69,8 @@ export const AddAlertRuleTemplateModal: FC<AddAlertRuleTemplateModalProps> = ({
               <HorizontalGroup justify="center" spacing="md">
                 <LoaderButton
                   data-qa="alert-rule-template-add-button"
+                  // TODO: fix LoaderButton types
```
[the remark this answers is not in the document]
Comment (lunaticusgreen) [reply public/app/features/integrated-alerting/components/AlertRuleTemplate/AddAlertRuleTemplateModal/AddAlertRuleTemplateModal.tsx]: Thanks, it was fixed in the latest version of platform package so this comment is no longer required

File: packages/grafana-ui/src/components/Slider/Slider.test.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (lunaticusgreen) [reply packages/grafana-ui/src/components/Slider/Slider.test.tsx:12]: It seems like it just something strange with Slider on grafana side, didn't want to spend a lot of time to investigate it if you don't mind I'll check changes of Slider component tests when will be working on an upgrade to grafana 7.4

**Review by fabio-silva — DISMISSED** [review]

**Review by atymchuk — APPROVED** [review]

**Pull request merged**